test/fixtures/simple_example/variables.tf

```diff
@@ -36,13 +36,13 @@ variable "names" {
 }
 
 variable "per_folder_admins" {
-  type        = list(string)
+  type        = map(string)
   description = "List of IAM-style members per folder who will get extended permissions."
-  default = [
-    "group:test-gcp-developers@test.infra.cft.tips",
-    "group:test-gcp-qa@test.infra.cft.tips",
-    "group:test-gcp-ops@test.infra.cft.tips",
-  ]
+  default = {
+    dev        = "group:gcp-ddt-developers@apszaz.com",
```

Comment:
We should keep the older names which exist for testing in our org.

Reply:
fixed in 3b439120a7ccb1c9de02ad0b099a653ccf73c493